.travis.yml

```diff
@@ -15,21 +15,27 @@ python:
   - "2.7"
 
 env:
-  - VERSION="8.0" LINT_CHECK="1"
-  - VERSION="8.0" ODOO_REPO="odoo/odoo" LINT_CHECK="0"
-  - VERSION="8.0" ODOO_REPO="OCA/OCB" LINT_CHECK="0"
+ global:
+ - VERSION="8.0" TESTS="1" LINT_CHECK="0"
+ - TRANSIFEX_USER='transbot@odoo-community.org'
+ - secure: CSf07Ubip0XXb19t6uBFhRMTlKFz7dGO2oxN5ibgObzjj+fncjdMsTqY0tIGT0/0+X9hWzDL4ZpY+1jhrHDgnDw9D2w81CZXC82DobPf+/pQ4MCHbZmldTqAE/APIzG76D6m77VB0yYGX+3LhNrpRKiDlWbLUFySQV5pe4py4R0=
+ 
+  matrix:
+  - LINT_CHECK="1"
+  - TRANSIFEX="1"
+  - TESTS="1" ODOO_REPO="odoo/odoo"
+  - TESTS="1" ODOO_REPO="OCA/OCB"
 
 virtualenv:
   system_site_packages: true
 
 install:
-  - git clone https://github.com/oca/maintainer-quality-tools.git ${HOME}/maintainer-quality-tools
-  #- git clone https://github.com/OCA/connector $HOME/connector -b ${VERSION}
+  - git clone --depth=1 https://github.com/oca/maintainer-quality-tools.git ${HOME}/maintainer-quality-tools
   - export PATH=${HOME}/maintainer-quality-tools/travis:${PATH}
   - travis_install_nightly
 
 script:
-  - travis_run_tests
+  - travis_after_tests_success
```

Comment:
It seems wrong, according to https://github.com/OCA/maintainer-quality-tools/blob/master/sample_files/.travis.yml#L78